### PR TITLE
Cleaning up dev ES instance ENV variables

### PR DESCRIPTION
**Pull request by vishnuchalla**
## Type of change

- [x] Refactor
- [ ] New feature
- [ ] Bug fix
- [x] Optimization
- [ ] Documentation Update

## Description
Removing dev ES instance ENV vars.
Pre-requisite: https://github.com/openshift/release/pull/57107

## Related Tickets & Documents
- Closes # https://issues.redhat.com/browse/PERFSCALE-3430

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
Through the CI tests

### CODE REVIEW

File: workloads/api-load/env.sh [hunk 1]
```diff
@@ -1,7 +1,6 @@
 # Common
 
 TEST_CLEANUP=${TEST_CLEANUP:-true}
-export ES_SERVER=${ES_SERVER:-https://search-perfscale-dev-chmf5l4sh66lvxbnadi4bznl3a.us-west-2.es.amazonaws.com:443}
 export ES_INDEX=ripsaw-api-load
```
Comment (krishvoor) [inline workloads/api-load/env.sh:4]: Let's have a placeholder with the default value `export ES_SERVER=${ES_SERVER:-""}`

**Review by krishvoor — COMMENTED** [review]
Minor question.

File: workloads/api-load/env.sh (hunk 1 above)
Comment (vishnuchalla) [reply workloads/api-load/env.sh:4]: The code flow and behavior would be the same. Any specific reason to add an empty string?

**Comment by jtaleric** [comment]
Could we say if ES_SERVER == ""; default to local index?

**Comment by vishnuchalla** [comment]
> Could we say if ES_SERVER == ""; default to local index?

IMHO both shouldn't be toggled that way. Local indexing should also happen only when a user opts for it.

**Review by dry923 — APPROVED** [review]
LGTM!

**Pull request merged**